libs/pmp-web/auth/data-access/src/lib/+state/auth.actions.ts

```diff
@@ -1,13 +1,17 @@
 import { HttpErrorResponse } from '@angular/common/http';
 import { Action } from '@ngrx/store';
 import { LoginPayload } from '@pimp-my-pr/pmp-web/auth/domain';
-import { LoginSuccessPayload } from '@pimp-my-pr/shared/domain';
+import { LoginSuccessPayload, GetUserInfoSuccessPayload } from '@pimp-my-pr/shared/domain';
+import { User } from '@pimp-my-pr/shared/domain';
 
 export namespace fromAuthActions {
   export enum Types {
     Login = '[Auth] Login',
     LoginFail = '[Auth] Login Fail',
     LoginSuccess = '[Auth] Login Success',
+    GetUserInfo = '[Auth] getUserInfo',
```

Comment:
Replace GetUserInfo with GetUserInfo and the rest of the place where UserInfo occurs. We are trying to name things like models on which they operate.